eventmesh-connector-plugin/eventmesh-connector-redis/src/test/java/org/apache/eventmesh/connector/redis/consumer/RedisConsumerTest.java

```diff
@@ -77,7 +77,7 @@ public void testSubscribe() throws Exception {
                 .withSubject("topic")
                 .withType(String.class.getCanonicalName())
                 .withDataContentType("text/plain")
-                .withData("data".getBytes())
+                .withData("data".getBytes(StandardCharsets.UTF_8))
```

Comment:
You need import the package `java.nio.charset.StandardCharsets` above, under the line 24